acceptance-tests/tests/src/test/resources/acceptanceTesting.security

```diff
@@ -2,5 +2,6 @@
 ### FOR ACCEPTANCE TESTING ONLY ###
 # Disable OS provided entropy
 securerandom.source=
+java.security.egd=
```

Comment:
I think since we already have securerandom.source set this won't have any effect

Reply:
yes indeed it does not correct anything . I will close this PR